agenta-backend/pyproject.toml

```diff
@@ -23,6 +23,7 @@ openai = "^0.27.8"
 langchain = "^0.0.256"
 odmantic = "^0.9.2"
 supertokens-python = "^0.15.1"
+sendgrid = "^6.10.0"
```

Comment:
should go to ee

Reply:
@aakrem sendgrid should go to ee?

Reply:
We can't move this dependency into ee. Below it, you'll see that supertokens is also here.